ajax/libs/js-popup/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "js-popup",
+  "filename": "js-popup.min.js",
+  "version": "1.0.0",
+  "description": "Simple popup messages for JavaScript/ReactJS.",
+  "keywords": [
+    "react",
+    "reactjs",
+    "popup",
+    "popup-window",
+    "popup-messages",
+    "popup-windows",
+    "popup-messaging",
+    "window",
+    "window-popup",
+    "popups",
+    "js-popup",
+    "component",
+    "JavaScript",
+    "box/popup window",
+    "box window",
+    "dialog window",
+    "dialog popup",
+    "dialog popup window"
```

Comment:
Here just suggestion for you.
There are many duplicated words such as `popup-window` and `popup-messages` which seemed not to let users be able to find the library more easily.
I tried some words in search engine like 
![example](https://cloud.githubusercontent.com/assets/13430892/24587719/e10a937e-17bb-11e7-8676-b07fbe429594.png)
Maybe you can simplify it.